.ci/dist-tag.groovy

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env groovy
+
+@Library('apm@current') _
+
+pipeline {
+  agent { label 'linux && immutable' }
+  environment {
+    REPO = "synthetics"
+    BASE_DIR = "src/github.com/elastic/${env.REPO}"
+    NPMRC_SECRET = 'secret/jenkins-ci/npmjs/elasticmachine'
+    TOTP_SECRET = 'totp/code/npmjs-elasticmachine'
+    NPM_PACKAGE = '@elastic/synthetics'
+    SLACK_CHANNEL = '#synthetics-user_experience-uptime'
+  }
```

Comment:
https://github.com/elastic/apm-pipeline-library/blob/3102b08c8aed84818d98406d91b40cc3401619ac/resources/JenkinsfileTemplate.groovy#L166-L168

```suggestion
    DIST_TAG = "${params.DIST_TAG}"
    VERSION = "${params.VERSION}"
  }
```

Reply:
This might be good to have so env variables for parameters for the very first build can have the right value